README.md

```diff
@@ -15,7 +15,7 @@
 
   * Install dependencies with `mix deps.get`
   * Create and migrate your database with `mix ecto.create && mix ecto.migrate`
-  * Install Node.js dependencies with `cd assets && npm install`
+  * Install Node.js dependencies with `cd assets && npm ci`
```

Comment:
This is exactly what I requested. `npm ci` removes the contents of the node_modules folder and reinstalls it, improving the dependency tree and avoiding errors and conflicts.